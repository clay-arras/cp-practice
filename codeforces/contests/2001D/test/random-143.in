1
9
23 5 21 3 5 16 19 15 21
