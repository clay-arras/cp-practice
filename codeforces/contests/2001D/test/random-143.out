7
23 3 5 16 19 15 21
